eduNEXT/ednx-saas-themes
# feat(FD-4): uptate favicon and logo image in cms and lms

## uptate favicon and logo image

This PR defines eduNEXT logo by default in Bragi.

- Add favicon to lms and cms

- Add studio openedx logo.

### How to test
- Use this strain with stack builder

```ruby
STRAIN_NAME: nuezdistro
STRAIN_TUTOR_VERSION: v14.0.5
DOCKER_IMAGE_OPENEDX_DEV: docker.io/ednxops/distro-edunext-edxapp-dev:nuez
DOCKER_IMAGE_OPENEDX: docker.io/ednxops/distro-edunext-edxapp:nuez
EDX_PLATFORM_REPOSITORY: https://github.com/eduNEXT/edunext-platform.git
EDX_PLATFORM_VERSION: ednx-release/nuez.master
CMS_HOST: studio.nutmeg.edunext.link
LMS_HOST: lms.nutmeg.edunext.link
DEV_PROJECT_NAME: nuezdistro_dev
LOCAL_PROJECT_NAME: nuezdistro_local
STRAIN_EXTRA_COMMANDS:
  - APP: "tutor"
    COMMAND: "distro enable-themes"
STRAIN_TUTOR_PLUGINS:
  - REPO: tutor-contrib-edunext-distro
    VERSION: v3.0.0
    EGG: tutor-contrib-edunext-distro==v3.0.0
    PACKAGE_NAME: distro
    PROTOCOL: ssh
  - REPO: tutor-forum
    VERSION: v14.0.0
    EGG: tutor-forum==v14.0.0
    ~PATH: overhangio~
    PACKAGE_NAME: forum
DISTRO_THEMES_NAME:
  - bragi
```

-` stack strain dev configure`
- `stack strain dev init`
- Now open a terminal in `env/build/openedx/themes/ednx-saas-themes` and chege the brach with `dcoa/add-logos-edunext`
- Go to LMS and check the favicon
- Go to CMS and check favicon and studio logo.

## Screenshots

LMS

![198744353-e256fcea-d6cb-402b-be11-0e8d5763ab5d](https://user-images.githubusercontent.com/91694045/204197360-eb7ee1f2-63ec-4bd5-b7d0-3872409380f7.png)

CMS
![198744922-e8186c7d-c488-4cc0-910c-50e22caea4d9](https://user-images.githubusercontent.com/91694045/204197382-90879482-41ea-424e-91c8-afe30cbf0077.png)

## Additional information

- [JIRA Issue](https://edunext.atlassian.net/browse/DS-299)
